source/opt/function.h

```diff
@@ -115,19 +119,24 @@ class Function {
   }
 
   // Runs the given function |f| on instructions in this function, in order,
-  // and optionally on debug line instructions that might precede them.
+  // and optionally on debug line instructions that might precede them and
+  // non-semantic instructions that succceed them.
```

Comment:
Small nit.  For the debug line instructions the "them" refers to the instructions in the function.  So it sounds like the non-semantic instructions that will be check are those that succeed every individual instruction in the function.  It might be more precise to say "non-semantic instruction that succeed the function."